opte/build.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Oxide Computer Company
```

Comment:
I haven't bothered to add any copyright/license to these basic build-related files. I have no strong opinion in that regard, it just didn't seem necessary to me. That said, as per RFD 224, I put OPTE under the MPL.

Reply:
ah, nice catch! I had copied the script from one of our other repos. I'll just remove this for now. We can do a bulk add as follow up later.